### ISSUE TITLE
A button/hotkey/chat command -  to allow timers that are synced across the group

### ISSUE THREAD

**Issue by Maskloss** [NeverScapeAlone/never-scape-alone#109]
Might not be a very much in demand feature but having a timer button for things where you want to have everyone do an action at the same time could be fun.
Could be that you want everyone to enter a boss room room at the same time.
( i am thinking here something like the Deadly Boss Mods series of World of warcraft mods that have boss pull timers.)

![image](https://user-images.githubusercontent.com/110988819/183975727-00cb49b7-387d-4d8e-ac66-5544b3fbeb8c.png)

Or a more longer term plan for a planned time to go off to Godwars with the group in 30 minutes time, giving people time to finish what they were doing before being at the door ready to go.
Could be a handy display too if it was able to show a timer like that to the group search so people can know how much longer till the group is at the door and ready. 
(may be best to allow a specific toggle for a timer that you want to show in the group search window to avoid any confusion)

**Comment by Ferrariic** [comment]
Keep coming with these great ideas. 

One issue that I forsee is that user clocks are not always set properly...(very frequently, they're wrong), however I think I could make this work :)

**Comment by Ferrariic** [comment]
I'll also likely only allow for a party leader to issue these commands

**Comment by Maskloss** [comment]
> I'll also likely only allow for a party leader to issue these commands

Yeah for sanity sake it would be best to keep it restricted to party leader

**Comment by Maskloss** [comment]
> Keep coming with these great ideas.
> 
> One issue that I forsee is that user clocks are not always set properly...(very frequently, they're wrong), however I think I could make this work :)

The timer could maybe pull from a universal timer rather then a direct user timer. im sure google has a timer that is accurate down to the millisecond. as for it being potentially off..well uh. maybe for bigger timers it would be best to have a little wiggle room. Could have a mouse over tool tip that mentions that timers may be off by X seconds/minutes and with a last pinged/checked thing.

**Comment by Ferrariic** [comment]
Sounds good! I'll see what I can do :)

**Comment by Ferrariic** [comment]
Actually, I could just send the time amount itself over the request, as broadcast on the api is synchronized — so then the clients themselves can tick down
